doc/manual/tuto.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: tutorial
+---
+
+# Managing users
+
+In this tutorial, you will learn how use Nickel to manage a list of users,
+and then export it as a YAML file for a program consuming it.
+
+## Step 1: Install nickel
+
+## Step 2: Think about the schema
+
+Our current YAML has different fields, such as `name`, `ssh-keys`,
+`is-admin` or `extra-groups`. To create a Nickel contract that produce
+what we want, and validates input, we need to think about the type of
+each field.
```

Comment:
```suggestion
`is-admin` or `extra-groups`. To create a Nickel contract that correctly
specifies the shape of this data, we first need to think about the
types of each field
```

I'm not sure it's entirely correct to say the contract "produces" the data we want - it really just does the specification & validation.